### ISSUE TITLE
[Feature Request] Another tab for non-seeded imported private keys.

### ISSUE THREAD

**Issue by kezimod** [spesmilo/electrum#4921]
I have to run two versions of electrum. One with a regular wallet and one for one by one Vanitygen private keys I've created.

**Comment by kezimod** [comment]
Yes, then please add an option to manage multiple wallets. Thanks

**Comment by SomberNight** [comment]
What do you mean by that?
You can have two windows (one for each wallet) open at the same time, in a single process.

**Comment by kezimod** [comment]
Implement a way to toggle between two wallets using tabs or a drop down menu in the same GUI in the same process.
